Show view sub-columns in information_schema.columns (#16165)
crate/crate: docs/sql/statements/create-view.rst

  been created, these columns will show up in subsequent queries on the view.
  It is generally recommended to avoid using ``*`` in view definitions.

 .. NOTE::

   If an object column is selected, sub-columns are visible in the

>    ``information_schema.columns`` table. New sub-columns, added via
>    ``ADD COLUMN`` or dynamically, are visible there as well. This applies both

Reviewer: ```suggestion
   ``information_schema.columns`` table. Added or dropped sub-columns at 
      the source relation are visible there as well. This applies both
```

Author: Dropped columns are not visible in the  `information_schema` (https://github.com/crate/crate/commit/e53f1f37888792cf1b2973a2e8208acda795f6c2)

They are exposed in pg_catalog (https://github.com/crate/crate/commit/9a2821de81ed5109e0d3d548ee82a28f95c569ea)

I removed "dropped".

Reviewer: Hm yes my suggestion was confusing. I wanted to express that if a column is dropped at the source relation, it won't show up as a view column anymore (columns are always resolved dynamically).

Author: Applied a suggestion from https://github.com/crate/crate/pull/16165#discussion_r1639681754, thanks!

>    relation are visible there as well. This applies both to views where ``*``

Reviewer: Maybe?
```suggestion
   relation are visible there  dropped columns won't show up there anymore as well while.
   This applies both to views where ``*``
```
